.omeroci/app-build

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+set +u
```

Comment:
I don't imagine any of the variables in this file do anything, right?

Reply:
no
that's cut &paste from another file
I will clean that up